packages/components/components/elvis-progress-linear/src/react/elvia-progress-linear.tsx

```diff
@@ -17,6 +18,17 @@ export const ProgressLinear: FC<ProgressLinearProps> = ({
   inlineStyle,
   ...rest
 }) => {
+  const lang = useLanguage();
+  const [computedAriaLabel, setAriaLabel] = useState('');
+  const [computedAriaValueText, setAriaValueText] = useState('');
```

Comment:
Consider renaming the state setter:
> The convention is to name state variables like [something, setSomething] using [array destructuring.](https://javascript.info/destructuring-assignment)

from: https://react.dev/reference/react/useState#reference

```suggestion
  const [computedAriaLabel, setComputedAriaLabel] = useState('');
  const [computedAriaValueText, setComputedAriaValueText] = useState('');
```